subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/parser/GradlePomModuleDescriptorParserBomTest.groovy

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.api.internal.artifacts.ivyservice.ivyresolve.parser
+
+import org.gradle.internal.component.external.descriptor.MavenScope
+
+import static org.gradle.api.internal.component.ArtifactType.MAVEN_POM
+
+class GradlePomModuleDescriptorParserBomTest extends AbstractGradlePomModuleDescriptorParserTest {
+
+    def "interprets dependencies declared in BOMs as optional dependencies"() {
+        given:
+        pomFile << """
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>group-a</groupId>
+    <artifactId>module-a</artifactId>
+    <version>1.0</version>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>group-b</groupId>
+                <artifactId>module-b</artifactId>
+                <version>1.0</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+</project>
+"""
+
+        when:
+        parsePom()
+
+        then:
+        def dep = single(metadata.dependencies)
+        dep.requested == moduleId('group-b', 'module-b', '1.0')
+        dep.scope == MavenScope.Compile
+        hasDefaultDependencyArtifact(dep)
+        dep.optional
+    }
+
+    def "interprets dependencies declared in parent BOMs as optional dependencies"() {
```

Comment:
⭕️ Confusing test name - There is no such thing as a "parent BOM". I guess you meant "a BOM that has a parent". We probably should add a similar test that makes sure that the `<dependencyManagement>` of the parent is ignored if the child has `<dependencies>`, just like in the test below.